src/components/Data/Documents/Views/TimeSeries.vue

```diff
@@ -213,25 +207,25 @@ export default {
   },
   methods: {
     updateChart() {
+      this.series = []
       this.chartOptions.colors = []
-      this.chart = []
-      const chartTitles = []
-      chartTitles.push(this.customDateField)
-
       for (const item of this.customNumberFields) {
         this.chartOptions.colors.push(item.color)
-        chartTitles.push(item.name)
       }
-      this.chart.push(chartTitles)
-
-      for (const doc of this.documents) {
-        const item = []
-        item.push(doc.content[this.customDateField])
-        for (const field of this.customNumberFields) {
-          item.push(_.get(doc.content, field.name, ''))
+      let series = []
+      for (const field of this.customNumberFields) {
+        let data = {
```

Comment:
```suggestion
        const data = {
```

Reply:
Also having `data.data` is a bit confusing :sweat_smile: 
IMHO `serie` is a better name for this variable